tests/schemas/webcompat-settings.json

```diff
@@ -0,0 +1,119 @@
+                "handlerStrategies"
+            ],
+            "properties": {
+                "state": {
+                    "$ref": "#/definitions/state"
+                },
+                "handlerStrategies": {
+                    "type": "object",
+                    "additionalProperties": false,
+                    "required": [
+                        "polyfill",
+                        "reflect",
+                        "undefined"
+                    ],
+                    "properties": {
+                        "reflect": {
+                            "type": "array",
+                            "items": {
+                                "type": "string"
+                            }
+                        },
+                        "polyfill": {
+                            "type": "array",
+                            "items": {
+                                "type": "string"
+                            }
+                        },
+                        "undefined": {
+                            "type": "array",
+                            "items": {
+                                "type": "string"
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "domains": {
+            "$ref": "#/definitions/domains"
+        }
+    },
+    "definitions": {
+        "state": {
+            "type": "string",
+            "enum": [
+                "enabled",
+                "disabled"
+            ]
+        },
+        "domains": {
+            "description": "List of domains with specific patch settings",
+            "type": "array",
+            "items": {
+                "title": "Domain",
+                "type": "object",
+                "additionalProperties": false,
+                "properties": {
+                    "domain": {
+                        "type": "string",
+                        "description": "Domain name"
+                    },
+                    "patchSettings": {
+                        "description": "List of operations to be applied on the settings for a specific domain",
+                        "type": "array",
+                        "items": {
+                            "type": "object",
+                            "title": "Patch Setting",
+                            "additionalProperties": false,
+                            "properties": {
+                                "op": {
+                                    "type": "string",
+                                    "description": "The operation to be performed"
+                                },
+                                "path": {
+                                    "type": "string",
+                                    "description": "The path of the setting to be patched"
+                                },
+                                "value": {
+                                    "type": "string",
```

Comment:
This is usually an object no?

Reply:
for my case, it's just a string. Perhaps we should make it more general when we extract this to be re-used